src/main/java/uk/gov/hmcts/reform/data/ingestion/camel/processor/FileReadProcessor.java

```diff
@@ -80,15 +81,11 @@ public void process(Exchange exchange) {
         //Check Stale OR Not existing file and exit run with proper error message
         if (getFileStatusInBlobContainer(fileName, schedulerTime).equals(STALE)) {
             exchange.getMessage().setHeader(IS_FILE_STALE, true);
-            auditService.auditException(exchange.getContext(), String.format(
-                "%s file with timestamp %s not loaded due to file stale error",
-                fileName,
-                DateFormatUtils.format(fileTimeStamp, "yyyy-MM-dd HH:mm:SS")));
-            return;
+            throw new RouteFailedException(String.format("%s file with timestamp %s not loaded due to file stale error",
+                fileName, DateFormatUtils.format(fileTimeStamp, "yyyy-MM-dd HH:mm:SS")));
         } else if (getFileStatusInBlobContainer(fileName, schedulerTime).equals(NOT_EXISTS)) {
-            auditService.auditException(exchange.getContext(), String.format(
-                "%s file is not exists in container", routeProperties.getFileName()));
-            return;
+            throw new RouteFailedException(String.format("%s file is not exists in container",
+                routeProperties.getFileName()));
```

Comment:
change error message to "%s file does not exist in azure storage account"

Reply:
done